.github/workflows/patpatbot.yml

```diff
@@ -0,0 +1,23 @@
+name: Review pattern documentation
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  run-patpatbot:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Run PatPatBot Action
+      uses: nicklem/patpatbot@main
```

Comment:
not all tools have `docs/` files versioned in the repo, for example Semgrep. they will only run the documentation generation process during the build process.
it could be more versatile to have patpatbot as a CLI that we could plug-in after the documentation generation, regardless of where it happens.